src/redisraft.c

```diff
@@ -15,18 +15,26 @@
 
 #include "redisraft.h"
 
-RedisModuleCtx *redis_raft_log_ctx = NULL;
+RedisModuleCtx *redisraft_log_ctx = NULL;
 
-int redis_raft_trace = TRACE_OFF;
-int redis_raft_loglevel = LOG_LEVEL_NOTICE;
+int redisraft_trace = TRACE_OFF;
+int redisraft_loglevel = LOG_LEVEL_NOTICE;
 
-const char *redis_raft_log_levels[] = {
+const char *redisraft_loglevels[] = {
     REDISMODULE_LOGLEVEL_DEBUG,
     REDISMODULE_LOGLEVEL_VERBOSE,
     REDISMODULE_LOGLEVEL_NOTICE,
     REDISMODULE_LOGLEVEL_WARNING
 };
 
+int redisraft_loglevel_enums[] = {
+    LOG_LEVEL_DEBUG,
+    LOG_LEVEL_VERBOSE,
+    LOG_LEVEL_NOTICE,
+    LOG_LEVEL_WARNING,
+    LOG_LEVEL_COUNT
+};
+
 RedisRaftCtx redis_raft = { 0 };
 static RedisRaftConfig config;
```

Comment:
why do we need to keep config as static? it's already a member of raft context...

Reply:
moved in to RedisRaftCtx